autoscheduler/frontend/src/components/HelpText.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Grid, Paper } from '@material-ui/core';
+
+const HelpText: React.SFC = function App() {
+  return (
+    <Grid
+      container
+      spacing={0}
+      direction="row"
+      justify="space-around"
+      alignItems="center"
+      style={{ minHeight: '65vh' }}
```

Comment:
I don't think we want this (if we do, it should be moved to the style of the `Paper` component in line 14), the height will automatically change to the size of the elements inside of it by default and I think it looks better that way

Reply:
I was using nested grids to keep all the text aligned, but using typography like you said in another comment will resolve both issues